app/api/helpers/permission_manager.py

```diff
@@ -225,3 +232,12 @@ def permission_manager(view, view_args, view_kwargs, *args, **kwargs):
         return permissions[args[0]](view, view_args, view_kwargs, *args, **kwargs)
     else:
         return ForbiddenError({'source': ''}, 'Access forbidden').respond()
+
+
+def has_access(access_level, **kwargs):
+    if access_level in permissions:
+        auth = permissions[access_level](lambda *a, **b: 100 is 100, (), {}, (), **kwargs)
```

Comment:
Actually lambda function has to return for any input here so I created an expression to return true but yeah I just tried writing "True" as expression and it worked. Please check.